packages/eslint-config-4catalyzer-react/rules.js

```diff
@@ -46,4 +46,42 @@ module.exports = {
   // JavaScript.
   'react/require-default-props': 'off',
   'react/destructuring-assignment': 'off',
+  'react/sort-comp': [
+    'error',
+    {
+      order: [
+        'static-methods',
+        'type-annotations',
+        'instance-variables',
+        'lifecycle',
+        '/^handle.+$/',
+        'everything-else',
```

Comment:
probably fine to keep the get/set bit? we do use those methods a bit. we can omit the `instance-methods` thing though if we want